### PR TITLE
Gcam macro

### PR DESCRIPTION
**Pull request by realxinzhao**
Two economics-related pages are updated to reflect GCAM-macro updates. They are mainly from the CMP. Links and references are added.

### CODE REVIEW

File: economy.md [hunk 1]
```diff
@@ -23,31 +23,178 @@ gcam-version: v6
...
+
+Another example of a useful cross-check is the equality between net exports of new, final goods and services and offsetting international capital transfers. That is, both the "$ROW$" column and "$ROW$" row must sum to zero.
+
+
+
+<img src="gcam-figs/GCAM_macro_SAM.png" alt="GCAM-macro (KLEM) Social Accounting Matrix" style="zoom:50%;" />
+
+Figure 2: GCAM-macro (KLEM) Social Accounting Matrix {: .fig}
+
+### Connect the Macro Equations to GCAM
```
Comment (gokuliyer) [inline economy.md]: I think this section can be removed.

File: economy.md [hunk 2]
```diff
@@ -23,31 +23,178 @@ gcam-version: v6
...
+
+#### Macro connection to GCAM
```
Comment (gokuliyer) [inline economy.md]: This one too.

File: economy.md [hunk 3]
```diff
@@ -23,31 +23,178 @@ gcam-version: v6
...
+
+Figure 2: GCAM-macro (KLEM) Social Accounting Matrix {: .fig}
+
+### Connect the Macro Equations to GCAM
+
+#### Macro connection to GCAM 
+
+All of the places in GCAM that use GDP to drive behavior still do, such as final demand, which may use an income elasticity, or the food demand model. Previously, they would look up the value from a GDP object. Given the GDP could change with every iteration of the solver, and the macro equations are simultaneously dependent on the outcome of such calculations within GCAM, we utilize $TRIAL$ markets to resolve the dependency. Therefore, the GCAM objects that need a GDP value can just look it up from this $TRIAL$ market at any time. Note: for convenience and added error checking, we include a $SectorUtil$ to look up the value.
+
+#### GCAM connection to Macro
```
Comment (gokuliyer) [inline economy.md]: This section can be removed

File: economy.md [hunk 4]
```diff
@@ -23,31 +23,178 @@ gcam-version: v6
...
+
+For any projected labor force and GDP pathway, total factor productivity values can be selected to reproduce that pathway. That is, the model can be calibrated to replicate reference scenario GDP values or to match any alternative future scenarios of GDP pathways, such as the Shared-Socioeconomic Pathways (SSPs) (IIASA, 2018). Alternatively, estimates of future labor supply and assumptions of total factor/labor/capital/energy productivity improvements can be used directly to determine future GDP outcomes. In all cases, these projects can subsequently be run in open GDP mode (the default option in GCAM v7). 
+
+Assumptions of savings and depreciation rates for future periods are exogenous inputs and can be readily changed. A simple regression model of the relationship between historical per capita GDP and savings rates was applied to adjust future savings rates from initial historical rates by region. Depreciation rates were held fixed at historical values, as they are more uniform across regions.
+
+
+
+### Calculating economic consequences of perturbations in GCAM
```
Comment (gokuliyer) [inline economy.md:160]: This is important and good to have here.

**Review by gokuliyer — CHANGES_REQUESTED** [review]

File: economy.md (hunk 1 above)
Comment (realxinzhao) [reply economy.md]: Done. Thanks!

File: economy.md (hunk 2 above)
Comment (realxinzhao) [reply economy.md]: Done. Thanks!

File: economy.md (hunk 3 above)
Comment (realxinzhao) [reply economy.md]: Done. Thanks!

**Review by gokuliyer — APPROVED** [review]

**Pull request merged**